docker-compose.yml

```diff
@@ -1,21 +1,17 @@
 version: '2'
 services:
   wordpress:
-    image: visiblevc/wordpress:latest
-    links:
-      - db
+    image: visiblevc/wordpress:php7.0
     ports:
       - 8080:80
       - 443:443
     volumes:
-      - ./data:/data
+      - data:/data
     environment:
       DB_NAME: wordpress
       DB_PASS: root
   db:
-    image: mysql:5.7 # or mariadb:10
-    ports:
-      - 3306:3306
```

Comment:
Two things here:
1. `PORTS` was removed because apparently in compose `v2` all services can access each other by default. So this is just redundant.
2. Switched to recommending `mariadb` because everybody seems to say that's the faster of the two, and also because I'm not a fan of oracle.